### DISCUSSION TITLE
百度搜索BUG

### DISCUSSION THREAD

**Discussion by Guojing6** [xiaoshuapp/explorer-xiaoshu#245] [Feedback]
在地址栏百度搜索插件正常加载

在百度页面编辑后搜索，插件不能加载 其他搜索引擎正常。
![Snipaste_2022-10-19_20-24-01](https://user-images.githubusercontent.com/43821055/196690575-04acfd80-8b28-4d50-93c9-793a2df2e93c.png)
![Snipaste_2022-10-19_20-24-47](https://user-images.githubusercontent.com/43821055/196690933-74d984dd-531f-4d47-82b7-9e2343ff0594.png)

**Comment by conghuahuadan** [comment]
同问题，通过百度首页搜索，无法触发插件

<img width="799" alt="image" src="https://user-images.githubusercontent.com/20864280/198782547-e13f9653-c79a-43d9-83c0-51f5e2990b5a.png">

**Comment by nexmoe** [comment]
目前仍在尝试修复，可以通过刷新使用

**Reply by qaq-qwq-qvq** [reply]
我昨天刚从edge插件商店下的还是有这种问题，应该是受百度自动生成的url影响解析过滤后应该可以解决
